src/layout/components/SideBar/component/People/People.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import Person from './components/Person/Person'
+import "../../../../../assets/styles/SideBar/People.scss"
+
+const People = () => {
+  return (
+    <div className='peoplecontainer'>
+      <Person url="https://api.dicebear.com/6.x/avataaars/svg?backgroundColor=c0aede&seed=p"
```

Comment:
move these persons into perosn array and use js map operator to iterate array and display them